### PR TITLE
#38 Fixes comment width on wider screens

### PR DESCRIPTION
**Pull request by Cleop**
#38 Fixes comment width on wider screens

### CODE REVIEW

File: assets/css/app.css [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 @media (min-width: 26em) {
   .comment-minw {
-    min-width: 91.45%;
+    min-width: 94.2%;
```
Comment (nelsonic) [inline assets/css/app.css]: oh the _precision_ ... 🙄 
excuse my ignorance, but could we use Tachyons' `w-90` for this? http://tachyons.io/docs/layout/widths/

**Comment by Cleop** [comment]
image
@nelsonic This is what w-90 gives us on wider screens (like @SimonLab's or my monitor at FocusHub) hence why we ended up with the precise measurement. Normally I'm not a fan either. The only other thing I could try would be flexbox if you think it's worth a shot?

**Comment by nelsonic** [comment]
@Cleop as long as we _capture_ ***why*** we are using an "exact" percentage, for width, it's fine.
But _perhaps_ we should _inform_ someone @Tachyons HQ in `case` it's a `bug` ... 🤔

**Comment by Cleop** [comment]
https://github.com/tachyons-css/tachyons/issues/221 - technically tachyons doesn't have min-widths at all...

**Comment by nelsonic** [comment]
Cool. thanks for linking to the issue. 👍 
bet you've never had `this` many comments on a _one-line_ PR. 😜

**Comment by tachyons** [comment]
I have nothing to do with this project nor @tachyons-css project. Please don't tag me here :-)

**Comment by nelsonic** [comment]
Yeah, sorry ... I did not mean to _incorrectly_ tag you. 🤦‍♂️ 
bet you get a _lot_ of that ... having a _similar_ username to a _super-popular_ project ... 🙄 
Won't happen again (_at least not from us!_) 👍

**Comment by tachyons** [comment]
No issues, just pinged to potentially avoid this in future :-). This is not first time I am getting notifications due to this typo

**Pull request merged**